src/app/app.component.scss

```diff
@@ -0,0 +1,4 @@
+$green: #04a87f;
+.background-green {
+    color: $green;
+}

```

Comment:
Create one common scss file for colors variable where you can declare any color variable and re-use that file throughout the app

Reply:
Created a common scss folder and added color.scss file.

Reply:
You have not imported the variables file